web/src/components/Both.js

```diff
@@ -55,6 +57,7 @@ export class Both extends Component {
 
         render() {
             const {handleSubmit, handleChange, handleChangeColor} = this
+            // CR * horizontal size
```

Comment:
This line crosses the entire horizon and then some. It's very long! Prettier can help automatically verticalize this line to be more readable. 🌮